frontend/src/scenes/session-recordings/player/SessionRecordingPlayer.tsx

```diff
@@ -220,6 +220,7 @@ export function SessionRecordingPlayer(props: SessionRecordingPlayerProps): JSX.
                                 <PlayerPersonMeta />
 
                                 <LemonSegmentedButton
+                                    data-attr="session-recording-player-view-choice"
```

Comment:
I think the idea as to why they don't support `data-attr` is that you add it to the individual buttons to get a more targeted click. Good example: https://github.com/PostHog/posthog/blob/e4594b44a0b6bf27170b2a7d4967f04e09f8b9b3/frontend/src/scenes/actions/ActionStep.tsx#L304-L330.

Would that work in this case instead?